### PR TITLE
create Services section and adjust navigation bar hot links

### LINKED ISSUE #20: Create Services section and adjust navigation bar hot links

### PR DESCRIPTION
**Pull request by IsaiasSantanaDosSantos**
Closed #20

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,10 +4,11 @@
     <meta charset="UTF-8" />
     <link rel="icon" type="image/jpg" href="/src/assets/img/Isaias-circle.png" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Santana Web Dev</title>
+    <title>Santana Web Developer</title>
```
Comment (IsaiasSantanaDosSantos) [inline index.html:7]: Right

**Review by IsaiasSantanaDosSantos — COMMENTED** [review]
All right

**Review by IsaiasSantanaDosSantos — COMMENTED** [review]
All right

File: src/main.jsx [hunk 2]
```diff
@@ -4,15 +4,25 @@ import ReactDOM from 'react-dom/client'
 import './styles/global.css'
 import { NavBar } from "./components/NavBar";
 import { Home } from './components/Home'
-import { About } from './components/About'
```
Comment (IsaiasSantanaDosSantos) [inline src/main.jsx:7]: All right

File: src/elements/MenuList/styled.js [hunk 3]
```diff
@@ -41,58 +41,57 @@ column-gap: 15px;
...
-        width: 100%;
-    }
-    .actived::after {
+    @media (max-width: 992px){
         position: absolute;
+        top: 63px;
+        left: 0;
         width: 100%;
-        top: 26px;
-    }
```
Comment (IsaiasSantanaDosSantos) [inline src/elements/MenuList/styled.js:95]: All right

File: src/elements/MenuList/styled.js [hunk 4]
```diff
@@ -41,58 +41,57 @@ column-gap: 15px;
         width: 100%;
         top: 26px;
     }
-`;
-
-export const ListMobile = styled.ul`
```
Comment (IsaiasSantanaDosSantos) [inline src/elements/MenuList/styled.js:46]: All right

File: src/elements/MenuList/styled.js [hunk 5]
```diff
@@ -2,7 +2,7 @@ import styled from 'styled-components'
 
 import* as colors from '../../config/colors'
 
-export const ListDestop = styled.ul`
```
Comment (IsaiasSantanaDosSantos) [inline src/elements/MenuList/styled.js:5]: All right

File: src/elements/MenuList/index.jsx [hunk 6]
```diff
@@ -1,262 +1,97 @@
...
-
-    const scrollToTop = () => {
-        scroll.scrollToTop();
-    };
-
     return (
         <>
-            {windowWidth > 992 ? (
-                <ListDestop>
-                    <li className='hoverTo'
```
Comment (IsaiasSantanaDosSantos) [inline src/elements/MenuList/index.jsx:73]: All right

File: src/elements/MenuList/index.jsx [hunk 7]
```diff
@@ -1,262 +1,97 @@
...
 
-import { ListDestop, ListMobile } from "./styled";
+import { MenuListContainer } from "./styled";
 
 export const MenuList = () => {
-    const [windowWidth, setWindowWidth] = useState(window.innerWidth);
     const [ activeLink, setActiveLink ] = useState("")
 
-    const handleSetActive = (to) => {
-        setActiveLink(to);
```
Comment (IsaiasSantanaDosSantos) [inline src/elements/MenuList/index.jsx:13]: All right

File: src/elements/MenuList/index.jsx [hunk 8]
```diff
@@ -1,262 +1,97 @@
...
+import { Link } from 'react-scroll';
 
-import { ListDestop, ListMobile } from "./styled";
+import { MenuListContainer } from "./styled";
 
 export const MenuList = () => {
-    const [windowWidth, setWindowWidth] = useState(window.innerWidth);
     const [ activeLink, setActiveLink ] = useState("")
 
-    const handleSetActive = (to) => {
```
Comment (IsaiasSantanaDosSantos) [inline src/elements/MenuList/index.jsx:12]: All right

File: src/elements/MenuList/index.jsx [hunk 9]
```diff
@@ -1,262 +1,97 @@
...
-import { useState, useEffect } from 'react';
-import { Link, Events, animateScroll as scroll } from 'react-scroll';
+import { useState } from 'react';
+import { Link } from 'react-scroll';
 
-import { ListDestop, ListMobile } from "./styled";
+import { MenuListContainer } from "./styled";
 
 export const MenuList = () => {
-    const [windowWidth, setWindowWidth] = useState(window.innerWidth);
```
Comment (IsaiasSantanaDosSantos) [inline src/elements/MenuList/index.jsx:9]: All right

File: src/elements/MenuList/index.jsx [hunk 10]
```diff
@@ -1,262 +1,97 @@
 /* eslint-disable no-unused-vars */
-/* eslint-disable react/no-unknown-property */
-import { useState, useEffect } from 'react';
-import { Link, Events, animateScroll as scroll } from 'react-scroll';
+import { useState } from 'react';
+import { Link } from 'react-scroll';
 
-import { ListDestop, ListMobile } from "./styled";
```
Comment (IsaiasSantanaDosSantos) [inline src/elements/MenuList/index.jsx:6]: All right

File: src/elements/MenuList/index.jsx [hunk 11]
```diff
@@ -1,262 +1,97 @@
 /* eslint-disable no-unused-vars */
-/* eslint-disable react/no-unknown-property */
```
Comment (IsaiasSantanaDosSantos) [inline src/elements/MenuList/index.jsx:2]: All right

File: src/components/Services/index.jsx [hunk 12]
```diff
@@ -0,0 +1,46 @@
+/* eslint-disable react/no-unescaped-entities */
+import { FaDesktop, FaPaintBrush, FaMobileAlt, FaCompass , FaRocket, FaUniversalAccess } from 'react-icons/fa'
```
Comment (IsaiasSantanaDosSantos) [inline src/components/Services/index.jsx:2]: All right

File: src/components/Resume/styled.js [hunk 13]
```diff
@@ -0,0 +1,43 @@
+import  styled  from "styled-components";
+
+import * as colors from '../../config/colors'
```
Comment (IsaiasSantanaDosSantos) [inline src/components/Resume/styled.js:3]: All right

File: src/components/Resume/index.jsx [hunk 14]
```diff
@@ -0,0 +1,15 @@
+/* eslint-disable react/no-unescaped-entities */
+// import { FaDesktop, FaPaintBrush, FaMobileAlt, FaCompass , FaRocket, FaUniversalAccess } from 'react-icons/fa'
+import { ResumeContent }from './styled';
```
Comment (IsaiasSantanaDosSantos) [inline src/components/Resume/index.jsx:3]: All right

File: src/components/PortFolio/styled.js [hunk 15]
```diff
@@ -0,0 +1,43 @@
+import  styled  from "styled-components";
+
+import * as colors from '../../config/colors'
```
Comment (IsaiasSantanaDosSantos) [inline src/components/PortFolio/styled.js:3]: All right

File: src/components/PortFolio/index.jsx [hunk 16]
```diff
@@ -0,0 +1,15 @@
+/* eslint-disable react/no-unescaped-entities */
+// import { FaDesktop, FaPaintBrush, FaMobileAlt, FaCompass , FaRocket, FaUniversalAccess } from 'react-icons/fa'
```
Comment (IsaiasSantanaDosSantos) [inline src/components/PortFolio/index.jsx:2]: All right

File: src/components/NavBar/index.jsx [hunk 17]
```diff
@@ -21,7 +24,16 @@ export const NavBar = () => {
     return (
         <Container>
             <div className='nav-container'>
-                <Logo/>
```
Comment (IsaiasSantanaDosSantos) [inline src/components/NavBar/index.jsx:24]: All right

File: src/components/Faq/styled.js [hunk 18]
```diff
@@ -0,0 +1,43 @@
+import  styled  from "styled-components";
```
Comment (IsaiasSantanaDosSantos) [inline src/components/Faq/styled.js:1]: All right

File: src/components/Faq/index.jsx [hunk 19]
```diff
@@ -0,0 +1,15 @@
+/* eslint-disable react/no-unescaped-entities */
+// import { FaDesktop, FaPaintBrush, FaMobileAlt, FaCompass , FaRocket, FaUniversalAccess } from 'react-icons/fa'
+import { FaqContent }from './styled';
+
```
Comment (IsaiasSantanaDosSantos) [inline src/components/Faq/index.jsx:4]: All right

File: src/components/Contact/styled.js [hunk 20]
```diff
@@ -0,0 +1,43 @@
+import  styled  from "styled-components";
+
+import * as colors from '../../config/colors'
+
+export const ContactContent = styled.div`
+width: 100%;
+padding: 65px 0;
+display: flex;
+flex-direction: column;
+align-items: center;
```
Comment (IsaiasSantanaDosSantos) [inline src/components/Contact/styled.js:10]: All right

File: src/components/Contact/index.jsx [hunk 21]
```diff
@@ -0,0 +1,15 @@
+/* eslint-disable react/no-unescaped-entities */
+// import { FaDesktop, FaPaintBrush, FaMobileAlt, FaCompass , FaRocket, FaUniversalAccess } from 'react-icons/fa'
+import { ContactContent }from './styled';
```
Comment (IsaiasSantanaDosSantos) [inline src/components/Contact/index.jsx:3]: All right

File: src/components/About/styled.js [hunk 22]
```diff
@@ -4,7 +4,7 @@ import * as colors from '../../config/colors'
 
 export const AboutContent = styled.div`
 width: 100%;
-padding: 40px 0;
```
Comment (IsaiasSantanaDosSantos) [inline src/components/About/styled.js:7]: All right

File: index.html [hunk 23]
```diff
@@ -4,10 +4,11 @@
...
     <link rel="preconnect" href="https://fonts.googleapis.com">
 <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
 <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap" rel="stylesheet">
+
```
Comment (IsaiasSantanaDosSantos) [inline index.html:11]: All right

**Review by IsaiasSantanaDosSantos — COMMENTED** [review]
All right

**Review by IsaiasDeveloper — APPROVED** [review]
All right

File: src/main.jsx (hunk 2 above)
Comment (IsaiasDeveloper) [reply src/main.jsx:7]: ok

File: src/elements/MenuList/styled.js (hunk 3 above)
Comment (IsaiasDeveloper) [reply src/elements/MenuList/styled.js:95]: OK

File: src/elements/MenuList/styled.js (hunk 4 above)
Comment (IsaiasDeveloper) [reply src/elements/MenuList/styled.js:46]: OK

File: src/elements/MenuList/styled.js (hunk 5 above)
Comment (IsaiasDeveloper) [reply src/elements/MenuList/styled.js:5]: OK

File: src/elements/MenuList/index.jsx (hunk 6 above)
Comment (IsaiasDeveloper) [reply src/elements/MenuList/index.jsx:73]: OK

File: src/components/Faq/index.jsx (hunk 19 above)
Comment (IsaiasDeveloper) [reply src/components/Faq/index.jsx:4]: OK

File: src/components/Contact/styled.js (hunk 20 above)
Comment (IsaiasDeveloper) [reply src/components/Contact/styled.js:10]: OK

File: src/components/Contact/index.jsx (hunk 21 above)
Comment (IsaiasDeveloper) [reply src/components/Contact/index.jsx:3]: OK

File: src/components/About/styled.js (hunk 22 above)
Comment (IsaiasDeveloper) [reply src/components/About/styled.js:7]: OK

File: index.html (hunk 23 above)
Comment (IsaiasDeveloper) [reply index.html:11]: OK

**Review by IsaiasDeveloper — APPROVED** [review]
OK

**Review by IsaiasSantanaDosSantos — COMMENTED** [review]
All right

**Comment by IsaiasSantanaDosSantos** [comment]
Resolved

**Pull request merged**